packages/block-editor/src/components/inserter/preview-panel.js

```diff
@@ -24,7 +24,12 @@ function InserterPreviewPanel( { item } ) {
 					<div className="block-editor-inserter__preview-content">
 						<BlockPreview
 							__experimentalPadding={ 16 }
-							viewportWidth={ 500 }
+							viewportWidth={
+								( hoveredItemBlockType.example &&
+									hoveredItemBlockType.example
+										.viewportWidth ) ||
+								500
```

Comment:
Potentially you could have written `hoveredItemBlockType.example?.viewportWidth ?? 500`

Reply:
Done